app/src/main/java/ch/epfl/sdp/blindly/main_screen/my_matches/MyMatchesAdapter.kt

```diff
@@ -16,10 +16,12 @@ import androidx.core.os.bundleOf
 import androidx.recyclerview.widget.RecyclerView
 import ch.epfl.sdp.blindly.R
 import ch.epfl.sdp.blindly.animations.RecordAnimations
-import ch.epfl.sdp.blindly.main_screen.chat.ChatActivity
+import ch.epfl.sdp.blindly.main_screen.my_matches.chat.ChatActivity
+import ch.epfl.sdp.blindly.main_screen.my_matches.match_profile.MatchProfileActivity
 import ch.epfl.sdp.blindly.main_screen.map.UserMapActivity
 
 private const val BUNDLE_MATCHED_UID_LABEL = "matchedId"
+private const val BUNDLE_MATCHED_USERNAME_LABEL = "username"
```

Comment:
I think it would be more coherent to use the const in the User class to pass in the bundle as key, so use USERNAME from the User instead

Reply:
Yeah I saw that there were a lot of duplicates. I'll work and that and make a new commit.